grails-app/controllers/au/org/ala/collectory/LicenceController.groovy

```diff
@@ -55,7 +55,7 @@ class LicenceController {
             return
         }
 
-        [licenceInstance: licenceInstance]
+        [licenceInsnce: licenceInstance]
```

Comment:
is this abbreviation intentional ?

Reply:
No. I'll check to see whether it's mysteriously covered by other things.